package.json

```diff
@@ -21,6 +21,7 @@
     "io-ts": "^1.1.1",
     "italia-ts-commons": "^1.15.0",
     "lodash": "^4.17.10",
+    "moment": "^2.22.1",
```

Comment:
Let's use [date-fns](https://github.com/date-fns/date-fns) instead, as it's designed to provide immutable data objects and a more functional approach